stemcell_builder/stages/base_ssh/apply.sh

```diff
@@ -80,6 +83,10 @@ elif [ "$(get_os_type)" == "ubuntu" ]; then
   sed "/^ *MACs/d" -i $chroot/etc/ssh/sshd_config
   echo 'MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-ripemd160-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,hmac-ripemd160,hmac-sha1' >> $chroot/etc/ssh/sshd_config
 
+elif [ "$(get_os_type)" == "opensuse" ]; then
```

Comment:
This `elif` will never get evaluated because the `if` on line 68 will evaluate to true when using opensuse.